src/main/resources/templates/chores/index.html

```diff
@@ -17,7 +17,7 @@
     </thead>
     <tr th:each="chore : ${chores}">
 <!--        <a th:href="'/chore/detail?choreId=' + ${chore.id}"-->
-        <td th:text="${chore.name}"></td>
+        <td> <a th:href="'/chore/detail?choreId=' + ${chore.id}" th:text="${chore.name}"></a></td>
```

Comment:
```suggestion
<<td> <a th:href="'/chores/detail?choreId=' + ${chore.id}" th:text="${chore.name}"></a></td>
```

Reply:
The directory is plural, which is why it wasn't working before even though the code is sound.